src/scene/text/TextStyle.ts

```diff
@@ -279,26 +275,12 @@ export class TextStyle extends EventEmitter<{
             this.fontSize = fullStyle.fontSize as number;
         }
 
-        if (style.dropShadow)
-        {
-            // is true / false
-            if (style.dropShadow instanceof Boolean)
-            {
-                if (style.dropShadow === true)
-                {
-                    this.dropShadow = {
-                        ...TextStyle.defaultTextStyle.dropShadow as TextDropShadow
-                    };
-                }
-            }
-            else
-            {
-                this.dropShadow = {
-                    ...TextStyle.defaultTextStyle.dropShadow as TextDropShadow,
-                    ...style.dropShadow as TextDropShadow
-                };
-            }
-        }
+        this._dropShadow = style.dropShadow ?? defaultStyle.dropShadow;
```

Comment:
no need for  the `?? defaultStyle.dropShadow;` bits
this happens above in line 259:

``` const fullStyle = { ...defaultStyle, ...style };```

Reply:
Yes! Good catch thanks.